Workbooks/Workloads/Onboard SQL Azure Insights/Onboard SQL Azure Insights.workbook

```diff
@@ -25,9 +36,12 @@
             "version": "KqlParameterItem/1.0",
             "name": "SQLServer",
             "label": "SQL Server",
-            "type": 5,
+            "type": 2,
             "isRequired": true,
-            "query": "Resources\n| where id =~ '{SqlDatabase}'\n| project SqlServer = extract('(.+/microsoft.sql/(managedinstances|servers)/.+)/databases/.+', 1, tolower(id))\n| project value = SqlServer, label = SqlServer, selected = true",
+            "multiSelect": true,
+            "quote": "\"",
+            "delimiter": ",\\r\\n    ",
+            "query": "Resources\n| where id in~ ({SqlDatabase})\n| project SqlServer = extract('.+/microsoft.sql/(managedinstances|servers)/(.+)/databases/.+', 2, tolower(id)), Db = extract('.+/microsoft.sql/(managedinstances|servers)/.+/databases/(.+)', 2, tolower(id))\n| project value = strcat('Server=', SqlServer, '.database.windows.net;Port=1433;database=', Db, ';User Id=<user>;Password=<pw>;app name=telegraf;log=1;'), label = SqlServer, selected = true",

```

Comment:
Doug found a way to do this without tolower(id). You use the (?i) to do case invariant comparisons in the regex.

Reply:
Fixed